### DISCUSSION TITLE
Edge-tts not working in WSL2 Ubuntu

### DISCUSSION THREAD

**Discussion by smcw66** [rany2/edge-tts#256] [General]
First time user here.

I installed edge-tts using  the CLI commands in WSL2, and i was able to get an output file of 'hello.mp3', but the mp3 files does not make any sound. 

not sure what to make of it yet 🤷

**Comment by WillJDunn** [comment]
I just tested it in my WSL2 Debian environment and it is working fine. Did you try opening or copying the generated hello.mp3 file in the host OS and playing it there? Might also be helpful to confirm/provide the specific commands you have run as well as any messages they provided.

In my case I executed the following (bash terminal):
```bash
sudo apt update
sudo apt upgrade pipx
pipx install edge-tts
pipx upgrade edge-tts
pipx ensurepath
edge-tts --text "Hello, world!" --write-media hello.mp3
explorer.exe .
```

At which point the WSL2 directory was opened in the Windows file explorer. I then opened the generated hello.mp3 file with the Windows media player & it worked fine.
